### DISCUSSION TITLE
How to handle Backhandler in Popup, Modal & Dialog

### DISCUSSION THREAD

**Discussion by KoushikMVN** [ant-design/ant-design-mobile#4580] [General]
How can i handle back when a `Popup` open? If i click back in andriod or IOS device `Popup` is close with back event trigger. How can i prevent back when `Popup` close?

**Comment by awmleer** [comment]
Do you mean clicking the physical back button on the devices?

**Reply by KoushikMVN** [reply]
Yes physical button

**Reply by awmleer** [reply]
The physical back button should trigger the navigation back of your browser. antd-mobile does handle that. It's strange that when you click the back button, Popup get closed.
